fetchyFetchBoi/fetchyFetchBoi.ts

```diff
@@ -4,18 +4,35 @@ import {
   getDataForDate,
   updateDataIfDifferentForDate,
 } from "../shared/dataRepository";
+import got from "got";
 
 const main = async () => {
-  const vaccineData = await getVaccineData();
+  try {
+    const vaccineData = await getVaccineData();
 
-  const existingData = await getDataForDate(vaccineData.dataValidAsAtNzTimeIso);
-  if (existingData == null) {
-    await createDataForDate(vaccineData.dataValidAsAtNzTimeIso, vaccineData);
-  } else {
-    await updateDataIfDifferentForDate(
-      vaccineData.dataValidAsAtNzTimeIso,
-      vaccineData
+    const existingData = await getDataForDate(
+      vaccineData.dataValidAsAtNzTimeIso
     );
+    if (existingData == null) {
+      await createDataForDate(vaccineData.dataValidAsAtNzTimeIso, vaccineData);
+    } else {
+      await updateDataIfDifferentForDate(
+        vaccineData.dataValidAsAtNzTimeIso,
+        vaccineData
+      );
+    }
+  } catch (error) {
+    console.error(error);
+
+    await got.post(process.env.DISCORD_WEBHOOK!, {
```

Comment:
Is this `!` expected to be here?

Reply:
yea 😅 

the environment var is of type `string | null` so it complains. 
This seemed like the easiest fix 😅

Reply:
ofc makes sense - I guess the other way would be to type check and not call the endpoint but yeh all gooooood